efficientdet/keras/efficientdet_keras.py

```diff
@@ -149,10 +148,10 @@ def build(self, feats_shape):
       self._add_wsm('ones')
     elif self.weight_method == 'channel_attn':
       num_filters = int(self.fpn_num_filters)
-      self._add_wsm(lambda shape, dtype, **kwargs: tf.ones(shape, dtype, **kwargs), num_filters)
+      self._add_wsm(lambda: tf.ones([num_filters]))
     elif self.weight_method == 'channel_fastattn':
       num_filters = int(self.fpn_num_filters)
-      self._add_wsm(lambda shape, dtype, **kwargs: tf.ones(shape, dtype, **kwargs), num_filters)
```

Comment:
Does it work on tf2?

Reply:
Good catch. I forgot to merge this change. Updated now.